llvm/lib/Target/AArch64/MCTargetDesc/AArch64TargetStreamer.cpp

```diff
@@ -151,3 +151,81 @@ llvm::createAArch64ObjectTargetStreamer(MCStreamer &S,
 MCTargetStreamer *llvm::createAArch64NullTargetStreamer(MCStreamer &S) {
   return new AArch64TargetStreamer(S);
 }
+
+void AArch64TargetStreamer::emitSubsection(
```

Comment:
The term "subsection" already has a meaning in MC, so this should be `emitAtributesSubsection`. Same comment applies to other names.